docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for installing on OpenShift Local
+
+OpenShift local requires a minimum amount of memory to complete the {ProductShortName} analysis. The more memory, the quicker the analysis is completed. The table below describes the various amounts of memory required by OpenShift local to complete the MTA binary analysis:
```

Comment:
Please use the correct style for the product name: "OpenShift Local". I would also add a hyperlink to the product page in at least one of the mentions: https://developers.redhat.com/products/openshift-local/overview